docs/training/apis-howto/api-pytorch-lightning-ug.rst

```diff
@@ -105,6 +105,12 @@ In addition, we also patched some ``LightningModule`` methods to make porting yo
    Determined will automatically log the metrics you return from ``training_step`` and
    ``validation_step`` to Tensorboard.
 
+.. note::
+
+   Determined environment images no longer contain PyTorch Lightning. To use PyTorch Lightning, you
```

Comment:
Suggestion ...
Determined environment images no longer include PyTorch Lightning. To use PyTorch Lightning, add a line similar to the following in the ``startup-hooks.sh`` script: